releng/plugins/org.polarsys.kitalpha.releng.targets/kitalpha.target

```diff
@@ -122,19 +122,19 @@
       <unit id="org.eclipse.emf.diffmerge.gmf.feature.source.feature.group" version="0.0.0"/>
       <unit id="org.eclipse.emf.diffmerge.egit.feature.feature.group" version="0.0.0"/>
       <unit id="org.eclipse.emf.diffmerge.egit.feature.source.feature.group" version="0.0.0"/>
-      <repository id="EMF-DiffMerge" location="https://download.eclipse.org/diffmerge/releases/0.12.0/emf-diffmerge-site/"/>
+      <repository id="EMF-DiffMerge" location="https://download.eclipse.org/diffmerge/stable/0.13.0-M3/emf-diffmerge-site/"/>

```

Comment:
There is a 0.13.0-M4 available, is it normal to use this one ?
Is the 0.13.0 release planned ?